docs/adr/0009-naming-conventions.md

```diff
@@ -0,0 +1,158 @@
+# Naming conventions
+
+Date: 2020-09-24
+
+## Status
+
+2020-12-02 Proposed
+
+## Component files
+
+### Principles
+
+- Keep it simple, reduce nesting
+- A `src/pages` folder will hold 'root' components and reflect the navigation structure (menu, routes) of the app
+- Components needed only by one 'root' component go into that root component's folder, no extra nesting
+- Components needed by more than one 'root' component go into `src/components`
+- Component folders, files, and auxiliary files, camel cased, starting with upper case
```

Comment:
The folder names are starting with lower case as I can see in the example below. what should it be? `kebab-case` or `PascalCase`? Is not usual to use `camelCase` when naming `folders`.